EyeDetection.test.py

```diff
@@ -0,0 +1,15 @@
+import cv2
+from EyeDetection import detectEyes
+
+cap = cv2.VideoCapture(0)
+
+while True:
+    _,img = cap.read()
+    eyes = detectEyes(img,True)
+    print(eyes)
```

Comment:
comment out/ remove  this as well

Reply:
This is just for testing purpose....will remove it in the next PR

Reply:
just push another commit it will reflect here and I'll merge the PR